src/HonzaBotner.Discord.Services/HonzaBotner.Discord.Services.csproj

```diff
@@ -6,4 +6,9 @@
     <ProjectReference Include="..\HonzaBotner.Services.Contract\HonzaBotner.Services.Contract.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Chronic.Core" Version="0.4.0" />
```

Comment:
As I am on mobile I can't check quickly, but I am not really sure I saw this nuget used.

Reply:
It's used for parsing natural language time expressions like `tomorrow at 12:00`